### PR TITLE
Add Entry: General Objective-C

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword List

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/objective-c/objective-c.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Objective-C'
+Description: 'Objective-C is a programming language that builds upon the C language by adding object-oriented programming with Smalltalk-style messaging. '
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'OOP'
```
Comment (Dusch4593) [inline content/general/concepts/objective-c/objective-c.md:8]:
```suggestion
  - 'iOS'
  - 'OOP'
```

File: content/general/concepts/objective-c/objective-c.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Objective-C'
+Description: 'Objective-C is a programming language that builds upon the C language by adding object-oriented programming with Smalltalk-style messaging. '
```
Comment (Dusch4593) [inline content/general/concepts/objective-c/objective-c.md:3]:
```suggestion
Description: 'Objective-C is a programming language that builds upon the C language by adding object-oriented programming with Smalltalk-style messaging.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished my review; just a few comments/suggestions from me.

**Comment by SSwiniarski** [comment]
@Dusch4593 I implemented your suggestions and added a line about current Objective-C history.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM for the next review! 👍🏻

**Review by HishamT — APPROVED** [review]
LGTM

**Comment by Dusch4593** [comment]
Looks good for merge! I made a small change in 9bbbce12d85e7aeb9e59cc20ed2b7332f6b45d0e where I made "Operating system" all lowercase.

**Pull request merged**